### PR TITLE
Fix favourite events endpoint

### PR DESCRIPTION
**Pull request by osammy**
# Description
Fix the favorite events issue.
## Type of change
Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)
## Change status
- [x] Complete, tested, ready to review and merge
# How Has This Been Tested?
Please describe steps taken to ensure this feature is functional and does not break other features:
# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have run the app using my feature and ensured that no functionality is broken
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by tolaked — APPROVED** [review]

File: resources/favEvents/favEvents.controller.js [hunk 1]
```diff
@@ -1,34 +1,64 @@
...
-  async findFav(userId) {
+  findFav(userId) {
+    return new Promise((resolve, reject) => {
     // Find all Fav Events for a user;
-    const favEventsIds = await FavEvent.find(
-      { userId }, (err, docs) => {
-        if (err) {
-          return false;
+      // eslint-disable-next-line consistent-return
+      FavEvent.find({ userId }, async (err, favEvents) => {
```
[the remark this answers is not in the document]
Comment (abidex4yemi) [reply resources/favEvents/favEvents.controller.js:10]: @osammy  please reviews this above message from `hound`

**Review by abidex4yemi — CHANGES_REQUESTED** [review]
Good job on this 💪please review the request below.

File: resources/favEvents/favEvents.controller.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (abidex4yemi) [reply resources/favEvents/favEvents.controller.js:10]: @osammy don't mind hound ci is not configured for arrow syntax that's why we have this error

**Review by abidex4yemi — COMMENTED** [review]
Looks good the error is due to hound ci not configured for arrow function.

**Pull request merged**